### PR TITLE
Add options to allow override of resources and replicas for grafana container pods

### PR DESCRIPTION
**Pull request by rajatvig**

### CODE REVIEW

File: grafana/grafana.libsonnet [hunk 1]
```diff
@@ -5,7 +5,7 @@ local k = import 'ksonnet/ksonnet.beta.3/k.libsonnet';
     namespace: 'default',
 
     versions+:: {
-      grafana: '5.2.4',
+      grafana: '5.4.3',
```
Comment (metalmatze) [inline grafana/grafana.libsonnet]: We can straight go to `6.0.1` if we touch this right now.
The downstream kube-prometheus project already uses that version.

Comment (rajatvig) [reply grafana/grafana.libsonnet]: Using that version now.

**Comment by brancz** [comment]
I don't understand why this needs to be directly in this project? Can't we just set this as patches as needed within our respective stack?

**Comment by rajatvig** [comment]
@brancz I agree that the version upgrade for Grafana can be performed in each stack, overriding the resources is rather complex.

**Comment by brancz** [comment]
I think at least the replica is very easy to override and doesn't belong in the `container` config object. Could we remove that part? I can live with the resource reqests/limits being in the config.

**Comment by rajatvig** [comment]
Agreed. Will create another PR to remove it.

**Comment by rajatvig** [comment]
https://github.com/brancz/kubernetes-grafana/pull/64

**Pull request merged**